### PR TITLE
Correct the import statement

### PR DESCRIPTION
**Pull request by Shivadp**

### CODE REVIEW

File: profiling/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 try:
     from cStringIO import StringIO
 except ImportError:
-    from StringIO import StringIO
+    from io import StringIO
```
Comment (char0n) [inline profiling/__init__.py]: Can we still retain support for python 2.x by using the following code ?
```python
try:
    from cStringIO import StringIO	
except ImportError:	
    try: 
       from StringIO import StringIO
    except importError:
       from io import StringIO

Comment (Shivadp) [reply profiling/__init__.py]: Its nice to have support. We can commit these changes

Comment (char0n) [reply profiling/__init__.py]: Can you  pls incorporate my comment into your PR ? Then we can merge immediately. Thank you

Comment (Shivadp) [reply profiling/__init__.py]: Changes incorporated. Please merge it

**Pull request merged**